js/script.js

```diff
@@ -1,66 +1,37 @@
-var nameto;
-var namefrom;
-var gender;
-var salutation;
-var datefrom;
-var date2;
-var namechild;
-var childgender;
-var childsalutation;
-var timego;
-var timestart;
-var childpronoun;
-var city;
-
-moment.lang("de").format('LL');
-moment.locale("de");
-
-function submit_data() {
-
-  var nameto = document.getElementById('nameto').value;
-  var namefrom = document.getElementById('namefrom').value;
-  var datefrom = document.getElementById('datefrom').value;
-  var timestart = document.getElementById('timestart').value;
-  var namechild = document.getElementById('namechild').value;
-  var timego = document.getElementById('timego').value;
-  var city = document.getElementById('city').value;
-  var gender = document.getElementById('gender').value;
-  if (gender === "women") {
-    var salutation = "Sehr geehrte Frau ";
-  } else {
-    var salutation = "Sehr geehrter Herr ";
-  }
-  var childgender = document.getElementById('childgender').value;
-  if (childgender === "daughter") {
-    var childsalutation = "meine Tochter ";
-  } else {
-    var childsalutation = "meinen Sohn ";
-  }
-  if (childgender === "daughter") {
-    var childpronoun = "Sie ";
-  } else {
-    var childpronoun = "Er ";
-  }
+moment.lang('de').format('LL')
+moment.locale('de')
 
-  var d = Date(Date.now());
-  a = d.toString();
-  datenow = moment(a).format('DD.MM.YYYY');
-  date2 = moment(datefrom).format('DD.MM.YYYY');
+function submit_data () {
+  const nameto = document.getElementById('nameto').value
+  const namefrom = document.getElementById('namefrom').value
+  const datefrom = document.getElementById('datefrom').value
+  const timestart = document.getElementById('timestart').value
+  const namechild = document.getElementById('namechild').value
+  const timego = document.getElementById('timego').value
+  const city = document.getElementById('city').value
+  const gender = document.getElementById('gender').value
 
+  const salutation = gender === 'woman' ? 'Sehr geehrte Frau' : 'Sehr geehrter Herr'
```

Comment:
In the HTML file stands 'women' instead of 'woman', so you have to change it here OR in the HTML, whatever you like more. When you change the HTML, please also replace 'men' with 'man' for convenience ;) 
```suggestion
  const salutation = gender === 'women' ? 'Sehr geehrte Frau' : 'Sehr geehrter Herr'
```